app/lib/features/space/dialogs/edit_space_sheet.dart

```diff
@@ -291,6 +314,16 @@ class _EditSpacePageConsumerState extends ConsumerState<EditSpacePage> {
     }
   }
 
+  // permission check
+  Future<bool> permissionCheck() async {
+    var member = await widget.space.getMyMembership();
+    bool p1 = member.canString('CanSetTopic');
+    bool p2 = member.canString('CanUpdateAvatar');
```

Comment:
actually checking for one is enough: if it doesn't have the one, it doesn't have the other either.

Reply:
Yes I know, just for safety purposes. If the spec updates in later time, this could get potential of bug and weird scenarios. I can change if its important.